tensorflow_model_optimization/g3doc/guide/clustering/clustering_with_keras.ipynb

```diff
@@ -0,0 +1,510 @@
+{
```

Comment:
<p>One of the general recommendations it to not have a code block do too many things at once. One of my prior comments suggested the headers that would make this clearer. We should go back to the comment and agree upon it and the structure. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>